src/wizard/wizard-form-header.tsx

```diff
@@ -1,17 +1,18 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import clsx from 'clsx';
-import React from 'react';
+import React, { MutableRefObject } from 'react';
 import { useDynamicOverlap } from '../app-layout/visual-refresh/hooks/use-dynamic-overlap.js';
 import styles from './styles.css.js';
 
 interface WizardFormHeaderProps {
   children: React.ReactNode;
   isMobile: boolean;
   isVisualRefresh: boolean;
+  __internalHeaderRef?: MutableRefObject<any> | null;
```

Comment:
Can we not use `any`?

If a better type can't be found, maybe we can go with `MutableRefObject<HTMLDivElement>` or `MutableRefObject<HTMLElement>`